src/core/packages/chrome/browser-internal/src/ui/header/header.tsx

```diff
@@ -92,8 +95,18 @@ export function Header({
 }: HeaderProps) {
   const [isNavOpen, setIsNavOpen] = useState(false);
   const [navId] = useState(htmlIdGenerator()());
+  const [hasBetaConfig, setHasBetaConfig] = useState(false);
   const headerActionMenuMounter = useHeaderActionMenuMounter(application.currentActionMenu$);
 
+  useLayoutEffect(() => {
```

Comment:
i think it could be cleaner if we'd use useObservable here 

const hasBeta$ = useMemo(() => observables.appMenu$.pipe(map(config => !!config && !!config.items && config.items.length > 0)) ?? EMPTY)